app/views/employees/_employee_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<main class="form-signin text-center">
+  <i class="fa-solid fa-user-plus fa-4x"></i>
+  <%= form_with(model: employee) do |form| %>
+    <div class="mb-3 mt-5 form-group">
+      <%= form.text_field :name, class: "form-control", placeholder: "Name"%>
+    </div>
+
+    <div class="mb-3 form-group">
+      <%= form.email_field :email, class: "form-control", placeholder: "Email" %>
+    </div>
+
+    <div class="mb-3 form-group">
+      <%= form.text_field :position, class: "form-control", placeholder: "Position" %>
+    </div>
+
+    <div class="mb-3 form-group">
+      <%= form.text_field :privateNumber, class: "form-control", placeholder: "Private number"%>
+    </div>
+
+    <div class="mb-3 form-group">
+      <%= form.text_field :statusActive, class: "form-control", placeholder:"Status"%>
+    </div>
+
+    <div class="mb-3 form-group">
+      <%= form.number_field :store_id, class: "form-control", placeholder:"Store ID"%>
+    </div>
+
+    <div class="actions">
+      <%= form.submit class: "btn btn-success btn-lg w-100"%>
+    </div>
+
+    <div class="mt-3">
+      <%= link_to "Back", employees_path,class: "btn btn-dark btn-lg w-100"%>
+    </div>
+  <% end %>
+</main>
```

Comment:
Add a new line at the end of this file
And same here with the standardized language